SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -292,7 +292,7 @@ bool USpatialActorChannel::ReplicateActor()
 	{		
 		if (bCreatingNewEntity)
 		{
-			// SAHIL: CHECK IF YOU PROPERLY FIX THIS
```

Comment:
Can you elaborate on this? Also, do we want a ticket?

Reply:
Going to be addressing it in the other PR